src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -522,6 +536,15 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   }
 }
 
+/*
+This rule changes the color and opacity of the text in a disabled v-select component.
+The :deep pseudo-element and !important flag are used to override Vuetify's default styles.
+*/
+:deep(.disabled-select.theme--light.v-select .v-select__selection--disabled) {
```

Comment:
this one is for:
<img width="1033" alt="image" src="https://github.com/bcgov/business-edit-ui/assets/16507223/12c4d3c2-23d1-44ff-8400-248daf62d474">

Reply:
Awesome comment! Now I know exactly what/why this is doing. Thanks!